scripts/upgrade/autobrr.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# autobrr upgrader
+# ludviglundgren 2021 for Swizzin
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+if [[ ! -f /install/.autobrr.lock ]]; then
+    echo_error "autobrr doesn't appear to be installed!"
+    exit 1
+fi
+
+_upgrade_autobrr() {
```

Comment:
I noticed this code is duplicate from the install script, this could be de-duped by defining the function once in `sources/functions/autobrr` and then using it both in the install script as well as here